packages/chart/src/CdcChartComponent.tsx

```diff
@@ -88,53 +87,54 @@ interface CdcChartProps {
   setSharedFilter?: (filter: any) => void
   setSharedFilterValue?: (value: any) => void
   dashboardConfig?: DashboardConfig
+  data: object[]
 }
+
 const CdcChart = ({
-  configUrl,
```

Comment:
How are we loading the configs from here? I'm not seeing this included in the loadConfig section.